src/tests/suite/index.ts

```diff
@@ -1,41 +1,31 @@
 import * as path from 'path';
 import * as Mocha from 'mocha';
-import * as glob from 'glob';
+import { glob } from 'glob';
```

Comment:
`glob` was previously an indirect dependency, even though we're using it explicitly.

These changes result from taking a dependency on the latest version, which provides a Promise-based async API.